__tests__/app.test.js

```diff
@@ -0,0 +1,5 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seed");
+const data = require("../db/data/test-data");
+const db = require("../connection.js");
```

Comment:
why has this file been made? Do you need a test file for this ticket? Make sure you don't add code beyond the scope of the task!